prov/verbs/src/fi_verbs.c

```diff
@@ -78,6 +78,7 @@
 		FI_SEND | FI_RECV | FI_REMOTE_READ | FI_REMOTE_WRITE | \
 		FI_REMOTE_SIGNAL)
 #define VERBS_MODE (FI_LOCAL_MR | FI_PROV_MR_ATTR)
+#define VERBS_TX_OP_FLAGS (FI_INJECT | FI_COMPLETION | FI_REMOTE_COMPLETE)
```

Comment:
I already merged this PR, but this is wrong.  There should not be any default op_flags defined by the provider.  These should only be set by the app.  FI_INJECT is doubly wrong, in that it forces the use of inject for all operations.